voice/bxml/verbs/speakSentence.md

```diff
@@ -40,22 +40,77 @@ To choose a specific voice by name, use the `voice` attribute.
 
 ### SSML Tags Supported:
 
-The table below lists the SSML tags that Bandwidth supports.  Full details about the tags can be found at https://www.w3.org/TR/2010/REC-speech-synthesis11-20100907/
-TODO: put descriptions here?
+The table below lists the SSML tags that Bandwidth supports.
+
+Full details about SSML tags can be found at:
+https://www.w3.org/TR/2010/REC-speech-synthesis11-20100907/
 
 | **tag**      | **description** |
 |:-------------|:----------------|
-| `<break>`    |                 |
-| `<emphasis>` |                 |
-| `<lang>`     |                 |
-| `<mark>`     |                 |
-| `<p>`        |                 |
-| `<phoneme>`  |                 |
-| `<prosody>`  |                 |
-| `<s>`        |                 |
-| `<say-as>`   |                 |
-| `<sub>`      |                 |
-| `<w>`        |                 |
+| `<break>`    | Adds a pause to the speech. You can specify the duration of the pause by using either the `strength` or the `time` attributes.<br><br> Attributes:<ul><li>`strength`: accepted values are: `none`, `x-weak`, `weak`, `medium` (default), `strong` or `x-strong`.</li><li>`time`: the duration of the pause in second or milliseconds (e.g. `1s` or `1000ms`) with a maximum value of 10 seconds.</li></ul> |
+| `<emphasis>` | Attributes:<ul><li>`level`: (optional) defines the strength of emphasis to be applied, accepted values are: `strong`, `moderate` or `reduced`.</li></ul> |
+| `<lang>`     | Specifies the natural language of the content.<br><br> Attributes:<ul><li>`xml:lang`: specifies the language, see accepted values below. |
+| `<p>`        | Adds a pause between paragraphs. |
+| `<phoneme>`  | Use phonetic pronunciation for specific text.<br><br> Attributes:<ul><li>`alphabet`: one of `ipa` or `x-sampa`.</li><li>`ph`: phonetic symbols.</li></ul> |
```

Comment:
should we add a link to a table of phonetic symbols?

Reply:
This is a good question, the ph symbols depend on which alphabet was chosen.
Now I noticed that the SSML standard mentions ipa but not others.
Amazon supports ipa/x-sampa, however, other vendors not necessarily support those, MS for instance supports ipa/sapi/ups, while IBM supports ipa/ibm.

Reply:
From the ssml docs:
> Synthesis processors SHOULD support a value for alphabet of "ipa", corresponding to Unicode representations of the phonetic characters developed by the International Phonetic Association [IPA].